File: 02-backend/epidemic-simulation/src/main/java/pl/polris/epidemicsimulation/entity/SimulationDetails.java
package pl.polris.epidemicsimulation.entity;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class SimulationDetails {

    // ----------- Fields -----------

    private SimulationParameters simulationParameters;

    private List<SimulationResult> simulationResultList;


    // ----------- Constructors -----------

    public SimulationDetails(SimulationParameters simulationParameters, List<SimulationResult> simulationResultList) {
        this.simulationParameters = simulationParameters;
        this.simulationResultList = simulationResultList;
    }
}

File: 03-frontend/epidemic-simulation-fe/src/app/common/simulation-details.ts
import { SimulationParameters } from "./simulation-parameters";
import { SimulationResult } from "./simulation-result";

export class SimulationDetails {

    simulationParameters: SimulationParameters;
    simulationResultList: SimulationResult[];


    constructor() {
        console.log("in simulaitonDetails constructor");
    }
}
